chromium_src/chrome/browser/ui/views/permissions/chip_controller.cc

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+// Workaround for startup crash when GetLocationBarView() here returns nullptr.
+#define BRAVE_CHIP_CONTROLLER_HIDE_CHIP if (GetLocationBarView())
```

Comment:
This is related to https://source.chromium.org/chromium/chromium/src/+/eeb6206faf566e6b73f2a62c7671e6b2aca7434b. This happens because we make additional calls to `LocationBarView::Update` from `BraveLocationBarView::Init` (via `Update` call) and then in `OnThemeChanged` while `GetLocationBarView()` still returns `nullptr`.

Reply:
I feel like we're doing something wrong here if this is happening. Can you create a follow-up issue to look at it and maybe @petemill can check it out?

Reply:
Issue created https://github.com/brave/brave-browser/issues/26653